controllers/storageclassclaim/storageclassclaim_controller.go

```diff
@@ -778,20 +824,48 @@ func (r *StorageClassClaimReconciler) reconcileCephFilesystemSubVolumeGroup() er
 	}
 
 	var cephFileSystemName string
+	var dataPoolName string
 	availableCephFileSystems := len(cephFilesystemList.Items)
 	if availableCephFileSystems == 0 {
 		return fmt.Errorf("no CephFileSystem found in the cluster")
 	}
-	if availableCephFileSystems > 1 {
-		klog.Warningf("More than one CephFileSystem found in the cluster, selecting the first one")
-	}
 
-	cephFileSystemName = cephFilesystemList.Items[0].Name
+	cephClients := map[string]string{
+		"provisioner": r.cephClientProvisioner.Name,
+		"node":        r.cephClientNode.Name,
+	}
```

Comment:
Why was this moved here? can we keep it in its original code line? I would prefer to define vars as close to usage as possible.

Reply:
its was left over when i stored subvolgroup name in the CR status, placing at the previous line